CustomRules.rb

```diff
@@ -0,0 +1,72 @@
+$dict = {3 => 'Fizz', 5 => 'Buzz', 7 => 'Bang', 11 => 'Bong', 13 => 'Fezz'}
+
+def rule_3(number, output)
+  if number % 3 == 0
+    output << $dict[3]
+  end
+  output
+end
+
+def rule_5(number,output)
+  if number % 5 == 0
+    output << $dict[5]
+  end
+  output
+end
+
+def rule_7(number,output)
+  if number % 7 == 0 && output != []
+    output << $dict[7]
+  end
+  output
+end

```

Comment:
Is this correct? If a number is a multiple of 7 and not a multiple of 3 or 5 it should still print `Bang` but I'm not sure your code does.

Reply:
The rule says 'If a number is a multiple of 7, print "Bang" instead of the number. For numbers which are multiples of seven and three / five, append Bang to what you'd have printed anyway.' Therefore what I understand is that it shouldn't change anything if it's a multiple of 7 but not a multiple of 3 or 5. And here output != [] means it's a multiple of 3 and/or 5.

Reply:
The first part 'If a number is a multiple of 7, print "Bang" instead of the number.' means that even if it's not a multiple of 3 or 5 it should still say Bang.

Reply:
Oh yeah sorry, I confused myself by only paying attention to the second sentence.